src/services/procosysIPOApi.ts

```diff
@@ -1,29 +1,81 @@
-import { AxiosInstance } from 'axios';
+import { objectToCamelCase } from '@equinor/procosys-webapp-components';
 import { IpoDetails, OutstandingIposType } from './apiTypes';
+import { StorageKey } from '@equinor/procosys-webapp-components';
+import { HTTPError } from './HTTPError';
 
 type ProcosysIPOApiServiceProps = {
-    axios: AxiosInstance;
+    baseURL: string;
+    callback?: (res: Response) => Response;
+};
+
+type GetOperationProps = {
+    abortSignal?: AbortSignal;
+    method: string;
+    headers: any;
+    responseType?: string;
 };
 
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-const procosysIPOApiService = ({ axios }: ProcosysIPOApiServiceProps) => {
+const procosysIPOApiService = (
+    { baseURL }: ProcosysIPOApiServiceProps,
+    token: string
+) => {
+    const callback = (resultObj: any, apiPath: string): string => resultObj;
+
+    const getByFetch = async (
+        url: string,
+        abortSignal?: AbortSignal,
+        additionalHeaders?: any
+    ): Promise<any> => {
+        const plantInStorage = window.localStorage.getItem(StorageKey.PLANT);
+        let headers;
+        if (plantInStorage !== 'undefined') {
```

Comment:
I think it should be  if (plantInStorage !== undefined) {, and not  if (plantInStorage !== 'undefined') {